coderd/coderdtest/coderdtest.go

```diff
@@ -238,6 +242,10 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 		options.Database, options.Pubsub = dbtestutil.NewDB(t)
 	}
 
+	if options.NotificationsEnqueuer == nil {
+		options.NotificationsEnqueuer = new(notiffake.FakeNotificationEnqueuer)
```

Comment:
Learning: Is it the same of `&notiffake.FakeNotificationEnqueuer{}`?

Reply:
Yes, it is just a different form of writing it 👍